index.html

```diff
@@ -33,7 +33,116 @@ <h2>I’m a software developer</h2>
+                        <li><button type="button">Ruby On Rails</button></li>
+                        <li><button type="button">CSS</button></li>
+                        <li><button type="button">Javascript</button></li>
+                        <li><button type="button">HTML</button></li>
+                    </ul>
+                    <button class="green-button" type="button">See Project</button>
+                </div>
+            </li>
+            <li class="card">
+                <div class="card-content">
+                    <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="card-buttons">
+                        <li><button type="button">Ruby On Rails</button></li>
+                        <li><button type="button">CSS</button></li>
+                        <li><button type="button">Javascript</button></li>
+                        <li><button type="button">HTML</button></li>
+                    </ul>
+                    <button class="green-button" type="button">See Project</button>
+                </div>
+            </li>
+            <li class="card">
+                <div class="card-content">
+                    <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="card-buttons">
+                        <li><button type="button">Ruby On Rails</button></li>
+                        <li><button type="button">CSS</button></li>
+                        <li><button type="button">Javascript</button></li>
+                        <li><button type="button">HTML</button></li>
+                    </ul>
+                    <button class="green-button" type="button">See Project</button>
+                </div>
+            </li>
+            <li class="card">
+                <div class="card-content">
+                    <h4 class="card-heading">Multi-Post Stories Gain+Glory</h4>
+                    <ul class="card-buttons">
+                        <li><button type="button">Ruby On Rails</button></li>
+                        <li><button type="button">CSS</button></li>
+                        <li><button type="button">Javascript</button></li>
+                        <li><button type="button">HTML</button></li>
+                    </ul>
+                    <button class="green-button" type="button">See Project</button>
+                </div>
+            </li>
+        </ul>
+     
+    </section>
+    <section class="section-3">
+        <div class="about">
+                <h2 class="about-title">About Me</h2>
+            <p class="p-two"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+            <button class="green-button margin-btn" type="button">Get My Resume</button>
+            <img class="about-image" src="img/illustration2.svg" alt="image">
```

Comment:
- [ ] **Please make sure that the decorations are background images instead of `img` tags.**

It is convention to always include decorations as background images. This way you can ensure that they scale well and never overlap other contents.

Reply:
I saw your review, but on Figma, they provided those two images without putting them in the same container, so it was impossible to put them in the same container so that's why I used an image tag because I couldn't use two backgrounds in one container.

Reply:
@Munsa1 Hey! I am sorry if I understood you incorrectly. As per my understanding, you are saying that because the two images are separate you cannot put them as background image? 
Well, there are few ways in which you can achieve this. One way I would do this is by using two different background images. Please refer to the code below. 
```
body {
  background-image: 
    url(image-one.jpg),
    url(image-two.jpg);
  background-position:
    top right, /* this positions the first image */
    bottom left; /* this positions the second image */
  background-repeat:
    no-repeat; /* this applies to both images */
}
```

Please don't hesitate to leave a comment if you need any further assistance.

Reply:
Thank you for the review, I have used the suggested code and it works.